sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs

```diff
@@ -37,12 +37,20 @@ protected QuestionAnsweringProjectsClient()
         {
         }
 
+        /// <summary> Initializes a new instance of QuestionAnsweringProjectsClient. </summary>
+        /// <param name="endpoint"> Supported Cognitive Services endpoint (e.g., https://&lt;resource-name&gt;.api.cognitiveservices.azure.com). </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public QuestionAnsweringProjectsClient(Uri endpoint, AzureKeyCredential credential) : this(endpoint, credential, new QuestionAnsweringClientOptions())
```

Comment:
I don't see a problem with this, but out of curiosity: why an overload instead of default parameter values like before?

Actually, one nit: passing a new options instance is not the same as the default arg of null. Perhaps it won't matter in practice, but I could see a client doing something different if options is null. Perhaps behavior should be retained?

Reply:
We don't have this overload before. What is the "default parameter values like before"?

> I could see a client doing something different if options is null.

In my understanding, this should be reached by customization. So this behavior could be retained.

Reply:
My point was that you're changing the behavior by passing in a non-null object instead of null, which was the previous default. But if this has been consistent with generated code, then this is fine.